### PR TITLE
Fix typos in Adanced Usage guide

### PR DESCRIPTION
**Pull request by adamjstewart**

### CODE REVIEW

File: docs/source/020_advanced.rst [hunk 1]
```diff
@@ -3,14 +3,14 @@ Advanced User Guide for Personal Modulefiles
...
+#. Easily change the version of applications or libraries under their own development.
 #. Better documentation for what software is available.
 
-You can create new version of some software and place it in your
-personal PATH and forget about it. At least when it is in a module, it 
-will be listed in the loaded modules it will also appear in the list of
-available software via ``module avail``
+You can create a new version of some software and place it in your
+personal ``PATH`` and forget about it. At least when it is in a module, it
+will be listed in the loaded modules. It will also appear in the list of
```
Comment (adamjstewart) [inline docs/source/020_advanced.rst]: Run-on sentence.

File: docs/source/020_advanced.rst [hunk 2]
```diff
@@ -20,12 +20,12 @@ module path::
...
 
-This will prepend ``/path/to/personal/modulefiles`` to the MODULEPATH
+This will prepend ``/path/to/personal/modulefiles`` to the ``MODULEPATH``
 environment variable. This means that any modulefiles defined here
-will be used instead of the system modules. 
+will be used instead of the system modules.
 
 Suppose that the user creates a directory called ``$HOME/modulefiles``
-and he wants a personal copy of the "git" package and he does the
+and they want a personal copy of the "git" package and they do the
```
Comment (adamjstewart) [inline docs/source/020_advanced.rst]: Genderless pronouns.

File: docs/source/020_advanced.rst [hunk 3]
```diff
@@ -44,15 +44,12 @@ modulefile for git one does: ::
...
+    local version = myModuleVersion()
+    local pkgName = myModuleName()
     local pkg     = pathJoin(home,"pkg",pkgName,version,"bin")
     prepend_path("PATH", pkg)
-    ^D  
+    ^D
 
-Starting first from the name: git/2.6.2.lua, modulefiles with the .lua
-extension are assumed to be written in lua and files without are
-assumed to be written in TCL.
```
Comment (adamjstewart) [inline docs/source/020_advanced.rst]: This paragraph is duplicated immediately below. I wasn't sure if I should get rid of this one or the next one. Honestly, both locations kind of feel awkward.

**Comment by rtmclay** [comment]
This has been merged in by hand as there were choices to be made.

**Pull request closed without merging**